surveys/promise-rejections/survey.md

```diff
@@ -0,0 +1,68 @@
+# Node.js Promise reject use case survey
+
+## Are you currently using Promises, async/await, a mix, or neither?
+
+Assume using as creating your own `new Promise` and `async function`
+
+ - [ ] Promises
+ - [ ] async/await
+ - [ ] Both
+ - [ ] Neither / not writing Promise-based code
+
+## Are you currently using `.catch()`, `try await catch`, a mix, or neither to handle rejections?
+
+When consuming Promises, async functions or thenables, which of the options below do you use to handle rejections?
+
+ - [ ] `.catch()`
+ - [ ] `try await catch`
+ - [ ] Both
+ - [ ] Neither / not writing Promise-based code
+
+## Do you use one or more of the following options as a global handler for Promise rejections?
+
+Leave empty if you use the default Node.js behavior
```

Comment:
maybe better to have an explicit option for "use the default behavior", otherwise you can't necessarily infer it based on "didn't answer this question"

Reply:
Absolutely. There are many who do not really have an understanding of what the default Node.js behavior actually is, even if they do use it. Alternatively, I would split this into two separate yes/no questions:

* Do you use `process.on('unhandledRejection')` as a global handler for Promise rejections?
* Do you use the `--unhandled-rejections` command-line flag as a global handler for Promise rejections?